### ISSUE TITLE
List of ORT Users and Consultancy Companies/Individuals

### ISSUE THREAD

**Issue by rettichschnidi** [oss-review-toolkit/ort#8381]
I am part of the team that is evaluating potential FOSS compliance solutions for my employer. As a developer and FOSS enthusiast, I am lobbying for ORT.

As with many FOSS offerings, when making the case for ORT, I stumbled on two things:

1. Proprietary solutions (Black Duck, Snyk, etc.) are very eager to send their sales representatives, promising the sky. For ORT however, I need to answer our questions (based on the Linux Foundations [An Open Guide To Evaluating Software Composition Analysis Tools](https://www.linuxfoundation.org/resources/publications/an-open-guide-to-evaluating-software-composition-analysis-tools)) myself.

2. To make ORT more attractive to our sourcing/non-dev-decision-makers, it would be very helpful to  point at commercial offerings that are willing to help with ORT adoption (and ongoing support).

Questions:

- Regarding 1), would a PR trying to document the answers to the questionnaire linked in 1) be appreciated?
- Regarding 2), any chance we could have a (curated) list of offerings somewhere on the ORT website? Maybe similar to what [Zephyr](https://www.zephyrproject.org/ecosystem-vendor-offerings/) is doing?

Meanwhile, those are commercial offerings of which I am aware of:

* [Double Open](https://www.doubleopen.org/)
  * Company
  * SasS, but no consulting
* EPAM:
  * Company
  * Consulting
* [Thomas Steenbergen](https://github.com/tsteenbe)?
  * Freelancer

**Comment by willebra** [comment]
Listing known users of ORT would not hurt either. Or perhaps those of the users that want to be included in the list.

**Comment by sschuberth** [comment]
> Listing known users of ORT would not hurt either.

See [ADOPTERS.md](https://github.com/oss-review-toolkit/ort/blob/main/ADOPTERS.md). And @mmurto wanted to work on https://github.com/oss-review-toolkit/ort/issues/7315 😉

**Comment by rettichschnidi** [comment]
> Listing known users of ORT would not hurt either. Or perhaps those of the users that want to be included in the list. As per the title of the issue.

Just updated the text a bit to reflect the title. And add incorporate the hint of @sschuberth.

**Comment by grnrs** [comment]
Just to add on this: I'm currently in a similar position as @rettichschnidi and had similar questions (especially 2 and 3 initially mentioned). 
Has something like a Landscape ever been considered for the Automated Compliance Tooling (ACT) project? (cf [OpenSSF](https://landscape.openssf.org/) or [CNCF](https://landscape.cncf.io/))?

**Comment by sschuberth** [comment]
> Has something like a Landscape ever been considered for the Automated Compliance Tooling (ACT) project?

I don't think so, but personally I regard ACT to be pretty much dead anyway, see e.g. its [news page](https://automatecompliance.org/news/) which was last updated in 2020.

Instead, I'd rather refer to the [OpenChain Reference Tooling Work Group's Landscape](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/), also see https://github.com/Open-Source-Compliance/Sharing-creates-value/tree/master/Tooling-Landscape.
